docs/config/README.md

```diff
@@ -24,19 +24,19 @@ disabled = true
 You can change default configuration file location with `STARSHIP_CONFIG` environment variable:
 
 ```sh
-export STARSHIP_CONFIG=~/.starship/config.toml
+export STARSHIP_CONFIG=~/.config/starship.toml
```

Comment:
I think this should be something like

```suggestion
export STARSHIP_CONFIG=~/example/non/default/path/starship.toml
```

To make it explicit that this is just an example.